logo.js

```diff
@@ -3693,4 +3693,47 @@ function LogoInterpreter(turtle, stream, savehook)
       throw new Error("Internal error: Unbound procedure");
     return value;
   });
+
+  //----------------------------------------------------------------------
+  //
+  // 9. Sound / Basic Music / Web Speech
+  //
+  //----------------------------------------------------------------------
+
+  var soundSeq = [];
+  var audioCtx = new (window.AudioContext || window.webkitAudioContext)();
```

Comment:
Please drop the prefixed version but add feature detection so the code works even if AudioContext is not available. That'll mean wrapping this entire chunk in something like `if ('AudioContext' in window) { ... }`